index.yaml

```diff
@@ -1,6 +1,26 @@
 apiVersion: v1
 entries:
   fusionauth:
+  - apiVersion: v2
```

Comment:
Correct.  The last commit to this file is a result of a tag event testing the actions workflow.  This will only be changed as part of the MR to the upstream repo.

Reply:
That's fine.. What I'm asking is for you to remove the changes to this file so that the index.yaml doesn't references releases we don't really have.

Reply:
Understood.  The helm index entry for `0.9.4` has been removed.